Lib/asyncio/unix_events.py

```diff
@@ -878,6 +878,73 @@ def __exit__(self, a, b, c):
         raise NotImplementedError()
 
 
+class PidfdChildWatcher(AbstractChildWatcher):
+    """Child watcher implementation using Linux's pid file descriptors.
+
+    This child watcher polls process file descriptors (pidfds) to await child
+    process termination. In some respects, PidfdChildWatcher is a "Goldilocks"
+    child watcher implementation. It doesn't require signals or threads, doesn't
+    interfere with any processes launched outside the event loop, and should
+    scale linearly with the number of subprocesses launched by the event
+    loop. The main disadvantage is that pidfds are specific to Linux, and only
+    work on recent (5.3+) kernels.
+    """
+
+    def __init__(self):
+        self._loop = None
+        self._callbacks = {}
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, a, b, c):
```

Comment:
This is mostly for consistency with the `__exit__` methods from the more recently added `ThreadedChildWatcher` and `MultiLoopChildWatcher`; not necessary, but I prefer to be more explicit with the `__exit__` arg names if that's okay.

```suggestion
    def __exit__(self, exc_type, exc_val, exc_tb):
```

Reply:
okay